bluemira/builders/EUDEMO/ivc/divertor.py

```diff
@@ -267,12 +278,15 @@ def _make_inner_baffle(
             inner_target_start = self._get_wire_end_with_largest(target, "x")
         return self.make_baffle(
             label=self.COMPONENT_INNER_BAFFLE,
-            start=np.array([x_lim, self.x_points[0].z]),
+            start=np.array([x_lim, z_lim]),
```

Comment:
Taking the x-point is not reliable for some reason. This amounts to the same thing and in fact gives us the flexibility to not build the divertor at the x-point z (which will be part of some future parameterisation)